cloud_functions/flagCryptominer.js

```diff
@@ -4,11 +4,20 @@ const utils = require('./utils')
 
 const run = utils.withErrorReporting((mixpanel, message) => {
   const userSubjectId = validate(message)
-  return promisify(mixpanel.people.set)(`google:${userSubjectId}`, { 'isCryptominer': true })
```

Comment:
`promisify` was giving me problems, I think because (as Brett pointed out) `this` gets mangled along the way. I wrote it out longhand. I hope I got it right.

Reply:
This looks right to me.

However, I'm curious about why `promisify` didn't work. From my reading of the client library, they weren't using `this`. I would have expected a problem of that nature to blow up the tests, too. What exactly was the issue?